kratos/geometries/quadrilateral_2d_8.h

```diff
@@ -828,20 +828,23 @@ template<class TPointType> class Quadrilateral2D8
      */
     Matrix& InverseOfJacobian( Matrix& rResult, IndexType IntegrationPointIndex, IntegrationMethod ThisMethod ) const override
```

Comment:
This is a square matrix, then can not be inverted in the standard way, there is a GeneralizedInverse, but this is costly